src/js/components/molecules/business-card/business-card.html

```diff
@@ -1,4 +1,11 @@
-<a class="business-card" :href="contentLink.url" rel="noopener" target="_blank" tabindex="0" v-if="title">
+<a class="business-card"
+   :href="contentLink.url"
+   rel="noopener"
+   target="_blank"
+   tabindex="0"
+   v-if="title"
```

Comment:
I wonder how this v-if="title" got in ;-) but it can be removed if you want. Title is a required prop.